grovedb/src/lib.rs

```diff
@@ -68,7 +64,7 @@ pub enum Error {
     CorruptedData(String),
 }
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
It is necessary to be `Clone` ?

Reply:
Resolved: changes to be made is subsequent pr